src/pages/listing/ListingPage.tsx

```diff
@@ -1,14 +1,31 @@
 import * as React from 'react';
+import { Col, Container, Row } from 'reactstrap';
 
 import ListingGallery from './ListingGallery';
 import ListingInformation from './ListingInformation';
 import BookingCard from './BookingCard';
+import BookingBar from './BookingBar';
 
 import { Listing } from 'networking/listings';
 
-const ListingPage = (listing: Listing) => (<>
-  <ListingGallery {...listing} />
-  <ListingInformation {...listing} />
-  <BookingCard {...listing} />
-</>);
+const ListingPage = (listing: Listing) => (
+  <>
```

Comment:
I think this can be <Fade>

Reply:
@kcvan  your comment got cut off

Reply:
ah thanks, was going to say this can be `<Fade>`